draft-ietf-quic-recovery.md

```diff
@@ -398,29 +419,23 @@ A packet sent on an RTO alarm MUST NOT be blocked by the sender's congestion
 controller. A sender MUST however count these bytes as additional bytes in
 flight, since this packet adds network load without establishing packet loss.
 
-## Generating Acknowledgements
+## Multiple Packet Number Space Optimizations {#optimizations}
 
-QUIC SHOULD delay sending acknowledgements in response to packets,
-but MUST NOT excessively delay acknowledgements of packets containing
-non-ack frames.  Specifically, implementaions MUST attempt to
-enforce a maximum ack delay to avoid causing the peer spurious
-timeouts.  The default maximum ack delay in QUIC is 25ms.
+There are cases where one may be able to gain recovery information from
+acknowledgements of packets in another packet number space, but they rely
+on complex assumptions about the peer’s processing and acknowledgement
+algorithms.  Even those are unable to quickly recover from cases such as
+losing the client's Initial, but receiving the 0-RTT packets.  Below is
+an optimization using coalesced packets and implicit acknowledgements.
```

Comment:
I don't see the optimization using coalesced packets and implicit acknowledgements.  If it's not immediately below this statement, perhaps add a forward reference?

Reply:
Oh, I meant to delete this graf too

Reply:
I moved Crypto Handshake Data below under generating acknowledgements.